docs/modules/ROOT/examples/tutorial/scripts/environment.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 export KUBECONFIG=""
-export RESTIC_REPOSITORY=s3:$(minikube service minio --url)/backups/
+export RESTIC_REPOSITORY=s3:$(minikube service list | grep 9001 | cut -d"|" -f 5 | tr -d "[:space:]")/backups/
```

Comment:
You could also use a port-forward here, to be a bit more consistent.

Reply:
ah true, I forgot the script...

Reply:
https://github.com/k8up-io/k8up/pull/666/commits/e12a60b13853dc56abc4e35635709d6e3a5c624d